.github/workflows/release.yml

```diff
@@ -15,9 +15,20 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-      - name: Set up docker image
-        run: docker build -t volta .
-        working-directory: ./ci/docker
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build and push
```

Comment:
For my own understanding, this is basically the same as running `docker build ...`, except that it also caches the built docker image so that future runs can just pull it down, rather than having to re-build?

Reply:
Yes, and it also supports other platforms than the host platform